### PR TITLE
fix: disable follow button if user follows  already

### LINKED ISSUE #51: Disable follow after follow request is accepted
I can follow Jordi again even though he has already accepted/declined my request.

image

### PR DESCRIPTION
**Pull request by chrsBa**

### CODE REVIEW

File: src/app/home/tabs/search/search.component.html [hunk 1]
```diff
@@ -15,15 +15,27 @@
...
+							<ion-button expand="block" fill="outline" (click)="follow(item.username)">
+								Follow
+							</ion-button>
+						</ion-card-content>
+					} @else {
+						<ion-card-content>
+							<ion-button expand="block" fill="outline" disabled="true"> Following </ion-button>
+						</ion-card-content>
+					}
+				</ion-card>
```
Comment (flyingbeat) [inline src/app/home/tabs/search/search.component.html:38]: lets add an @empty{ section if no users were found

File: src/app/home/tabs/search/search.component.html [hunk 2]
```diff
@@ -15,15 +15,27 @@
...
-			@for(item of searchState.searchResults(); track item.username) {
-			<ion-card>
-				<ion-card-header>
-					<ion-card-title class="hover-pointer" (click)="router.navigate(['/home/profiles/' + item.username])">{{ item.username }}</ion-card-title>
-				</ion-card-header>
-				<ion-card-content>
-					<ion-button expand="block" fill="outline" (click)="follow(item.username)"> Follow </ion-button>
-				</ion-card-content>
-			</ion-card>
+			@for (item of searchState.searchResults(); track item.username) {
```
Comment (flyingbeat) [inline src/app/home/tabs/search/search.component.html:18]: lets display a loading spinner if the users are loading

**Review by flyingbeat — CHANGES_REQUESTED** [review]

**Comment by flyingbeat** [comment]
@chrsBa I have reworked the whole network / search pages for issue #66 on the corresponding branch

I copied your changes from this PR to there. I think we can close this PR (without merging) and continue on the other branch if you are ok with it.

**Comment by larsuzh** [comment]
changes moved to branch #66

**Pull request closed without merging**